Src/UIComponents/Dashboard/Dashboard.css

```diff
@@ -0,0 +1,151 @@
+* {
+    padding: 0;
+    margin: 0;
+}
```

Comment:
Maybe this could be included in app.css instead of specific component styling file? It will affect all other components (without any adverse effects) so I think such a global change should be stored there.